_documentation/subscriptions/subscription-groups.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Subscription groups / tags / categories
```

Comment:
This appears also in the breadcrumb, and it's a bit confusing with the slashes.
```suggestion
title: Subscription tags (aka groups or categories)
```
Ideally I would leave out 'groups' and 'categories' altogether, but I realise it'll help people find this article.